### ISSUE TITLE
ZeroDivisionError after website created

### ISSUE THREAD

**Issue by yaroslav-qlicks** [experius/SeoSnap#20]
Hi,

I'm trying to check how seosnap working
and after I add website, I got ZeroDivisionError
I tried to use development and release versions: same error

![image](https://user-images.githubusercontent.com/80677410/111490367-a40ff580-8743-11eb-8865-7cca02765cf5.png)
```Environment:


Request Method: GET
Request URL: http://0.0.0.0/seosnap/website/

Django Version: 3.0.7
Python Version: 3.7.10
Installed Applications:
['django.contrib.admin',
 'django.contrib.auth',
 'django.contrib.contenttypes',
 'django.contrib.sessions',
 'django.contrib.messages',
 'django.contrib.staticfiles',
 'rest_framework',
 'guardian',
 'seosnap']
Installed Middleware:
['django.middleware.security.SecurityMiddleware',
 'django.contrib.sessions.middleware.SessionMiddleware',
 'django.middleware.common.CommonMiddleware',
 'django.middleware.csrf.CsrfViewMiddleware',
 'django.contrib.auth.middleware.AuthenticationMiddleware',
 'django.contrib.messages.middleware.MessageMiddleware',
 'django.middleware.clickjacking.XFrameOptionsMiddleware']


Template error:
In template /usr/local/lib/python3.7/site-packages/django/contrib/admin/templates/admin/base.html, error at line 62
   division by zero
   52 :             {% endblock %}
   53 :         </div>
   54 :         {% endif %}
   55 :         {% endblock %}
   56 :         {% block nav-global %}{% endblock %}
   57 :     </div>
   58 :     <!-- END Header -->
   59 :     {% block breadcrumbs %}
   60 :     <div class="breadcrumbs">
   61 :     <a href="{% url 'admin:index' %}">{% trans 'Home' %}</a>
   62 :     {% if t itle %} &rsaquo; {{  title }}{% endif %}
   63 :     </div>
   64 :     {% endblock %}
   65 :     {% endif %}
   66 : 
   67 :     {% block messages %}
   68 :         {% if messages %}
   69 :         <ul class="messagelist">{% for message in messages %}
   70 :           <li{% if message.tags %} class="{{ message.tags }}"{% endif %}>{{ message|capfirst }}</li>
   71 :         {% endfor %}</ul>
   72 :         {% endif %}


Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/django/db/models/options.py", line 581, in get_field
    return self.fields_map[field_name]

During handling of the above exception ('cache_quality'), another exception occurred:
  File "/usr/local/lib/python3.7/site-packages/django/contrib/admin/utils.py", line 262, in lookup_field
    f = _get_non_gfk_field(opts, name)
  File "/usr/local/lib/python3.7/site-packages/django/contrib/admin/utils.py", line 293, in _get_non_gfk_field
    field = opts.get_field(name)
  File "/usr/local/lib/python3.7/site-packages/django/db/models/options.py", line 583, in get_field
    raise FieldDoesNotExist("%s has no field named '%s'" % (self.object_name, field_name))

During handling of the above exception (Website has no field named 'cache_quality'), another exception occurred:
  File "/usr/local/lib/python3.7/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/usr/local/lib/python3.7/site-packages/django/core/handlers/base.py", line 145, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/usr/local/lib/python3.7/site-packages/django/core/handlers/base.py", line 143, in _get_response
    response = response.render()
  File "/usr/local/lib/python3.7/site-packages/django/template/response.py", line 105, in render
    self.content = self.rendered_content
  File "/usr/local/lib/python3.7/site-packages/django/template/response.py", line 83, in rendered_content
    return template.render(context, self._request)
  File "/usr/local/lib/python3.7/site-packages/django/template/backends/django.py", line 61, in render
    return self.template.render(context)
  File "/usr/local/lib/python3.7/site-packages/django/template/base.py", line 171, in render
    return self._render(context)
  File "/usr/local/lib/python3.7/site-packages/django/template/base.py", line 163, in _render
    return self.nodelist.render(context)
  File "/usr/local/lib/python3.7/site-packages/django/template/base.py", line 936, in render
    bit = node.render_annotated(context)
  File "/usr/local/lib/python3.7/site-packages/django/template/base.py", line 903, in render_annotated
    return self.render(context)
  File "/usr/local/lib/python3.7/site-packages/django/template/loader_tags.py", line 150, in render
    return compiled_parent._render(context)
  File "/usr/local/lib/python3.7/site-packages/django/template/base.py", line 163, in _render
    return self.nodelist.render(context)
  File "/usr/local/lib/python3.7/site-packages/django/template/base.py", line 936, in render
    bit = node.render_annotated(context)
  File "/usr/local/lib/python3.7/site-packages/django/template/base.py", line 903, in render_annotated
    return self.render(context)
  File "/usr/local/lib/python3.7/site-packages/django/template/loader_tags.py", line 150, in render
    return compiled_parent._render(context)
  File "/usr/local/lib/python3.7/site-packages/django/template/base.py", line 163, in _render
    return self.nodelist.render(context)
  File "/usr/local/lib/python3.7/site-packages/django/template/base.py", line 936, in render
    bit = node.render_annotated(context)
  File "/usr/local/lib/python3.7/site-packages/django/template/base.py", line 903, in render_annotated
    return self.render(context)
  File "/usr/local/lib/python3.7/site-packages/django/template/loader_tags.py", line 150, in render
    return compiled_parent._render(context)
  File "/usr/local/lib/python3.7/site-packages/django/template/base.py", line 163, in _render
    return self.nodelist.render(context)
  File "/usr/local/lib/python3.7/site-packages/django/template/base.py", line 936, in render
    bit = node.render_annotated(context)
  File "/usr/local/lib/python3.7/site-packages/django/template/base.py", line 903, in render_annotated
    return self.render(context)
  File "/usr/local/lib/python3.7/site-packages/django/template/loader_tags.py", line 62, in render
    result = block.nodelist.render(context)
  File "/usr/local/lib/python3.7/site-packages/django/template/base.py", line 936, in render
    bit = node.render_annotated(context)
  File "/usr/local/lib/python3.7/site-packages/django/template/base.py", line 903, in render_annotated
    return self.render(context)
  File "/usr/local/lib/python3.7/site-packages/django/template/loader_tags.py", line 62, in render
    result = block.nodelist.render(context)
  File "/usr/local/lib/python3.7/site-packages/django/template/base.py", line 936, in render
    bit = node.render_annotated(context)
  File "/usr/local/lib/python3.7/site-packages/django/template/base.py", line 903, in render_annotated
    return self.render(context)
  File "/usr/local/lib/python3.7/site-packages/django/contrib/admin/templatetags/base.py", line 33, in render
    return super().render(context)
  File "/usr/local/lib/python3.7/site-packages/django/template/library.py", line 214, in render
    _dict = self.func(*resolved_args, **resolved_kwargs)
  File "/usr/local/lib/python3.7/site-packages/django/contrib/admin/templatetags/admin_list.py", line 342, in result_list
    'results': list(results(cl)),
  File "/usr/local/lib/python3.7/site-packages/django/contrib/admin/templatetags/admin_list.py", line 318, in results
    yield ResultList(None, items_for_result(cl, res, None))
  File "/usr/local/lib/python3.7/site-packages/django/contrib/admin/templatetags/admin_list.py", line 309, in __init__
    super().__init__(*items)
  File "/usr/local/lib/python3.7/site-packages/django/contrib/admin/templatetags/admin_list.py", line 232, in items_for_result
    f, attr, value = lookup_field(field_name, result, cl.model_admin)
  File "/usr/local/lib/python3.7/site-packages/django/contrib/admin/utils.py", line 271, in lookup_field
    value = attr(obj)
  File "/code/seosnap/admin/website.py", line 73, in cache_quality
    percentage = (cached_pages / all_pages) * 100;

Exception Type: ZeroDivisionError at /seosnap/website/
Exception Value: division by zero
```

**Comment by ghost** [comment]
Issue confirmed
https://github.com/experius/SeoSnap-Dashboard/blob/bebe63ecd8305dd11f2725efb50fbefd14d75e31/seosnap/admin/website.py#L73

Tmp fix

` percentage = ((cached_pages + 1) / (all_pages + 1)) * 100;`

**Comment by egordm** [comment]
Hi @yaroslav-qlicks,

You are indeed right. The fix is available in 0.4.1
